core/server/data/schema/schema.js

```diff
@@ -10,20 +10,31 @@
 module.exports = {
     newsletters: {
         id: {type: 'string', maxlength: 24, nullable: false, primary: true},
-        name: {type: 'string', maxlength: 191, nullable: false},
+        name: {type: 'string', maxlength: 191, nullable: false, unique: true},
         description: {type: 'string', maxlength: 2000, nullable: true},
+        slug: {type: 'string', maxlength: 191, nullable: false, unique: true},
         sender_name: {type: 'string', maxlength: 191, nullable: false},
-        sender_email: {type: 'string', maxlength: 191, nullable: false, validations: {isEmail: true}},
-        sender_reply_to: {type: 'string', maxlength: 191, nullable: false, validations: {isEmail: true}},
+        sender_email: {type: 'string', maxlength: 191, nullable: true},
+        sender_reply_to: {type: 'string', maxlength: 191, nullable: false, defaultTo: 'newsletter', validations: {isIn: ['newsletter', 'support']}},
         status: {type: 'string', maxlength: 50, nullable: false, defaultTo: 'active'},
-        recipient_filter: {
-            type: 'text',
-            maxlength: 1000000000,
+        visibility: {
+            type: 'string',
+            maxlength: 50,
             nullable: false,
-            defaultTo: ''
+            defaultTo: 'members'
         },
-        subscribe_on_signup: {type: 'bool', nullable: false, defaultTo: false},
-        sort_order: {type: 'integer', nullable: false, unsigned: true, defaultTo: 0}
+        subscribe_on_signup: {type: 'bool', nullable: false, defaultTo: true},
+        sort_order: {type: 'integer', nullable: false, unsigned: true, defaultTo: 0},
```

Comment:
Should this be unique? If we default to zero we might accidentally change the default newsletter

Reply:
🤔 We don't have any pattern for enforcing uniqueness on `sort_order`, i think it might make more guard against this in the model layer/business logic